config/settings/base.py

```diff
@@ -113,7 +113,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/Phoenix'
```

Comment:
This will fix our view, but will this mess up dates for international workshops? Is a timezone something that can be put into our Workshop date models?